src/pages/LogOutPreviousUserPage.tsx

```diff
@@ -22,10 +28,12 @@ type LogOutPreviousUserPageProps = LogOutPreviousUserPageOnyxProps & StackScreen
 // out if the transition is for another user.
 //
 // This component should not do any other navigation as that handled in App.setUpPoliciesAndNavigate
-function LogOutPreviousUserPage({session, route}: LogOutPreviousUserPageProps) {
+function LogOutPreviousUserPage({session, route, account}: LogOutPreviousUserPageProps) {
+    const initUrl = useContext(InitialUrlContext);
     useEffect(() => {
-        Linking.getInitialURL().then((transitionURL) => {
+        Linking.getInitialURL().then((url) => {
```

Comment:
**suggestion:** update the `InitialURLContext.Provider` to populate:

- first from the `url` passed from OldApp
- from `Linking.getInitialURL` as a fallback

That way, it can be used as a single source of truth for "where we're initially trying to go", and we don't have to keep track of both that _and_ `Linking.getInitialURL`